.github/workflows/test-external.yml

```diff
@@ -67,5 +73,5 @@ jobs:
         run: python -m pip install -e ./halmos
 
       - name: Test external repo
-        run: ${{ matrix.cmd }} -v -st --solver-timeout-assertion 0 --solver-threads 2
+        run: ${{ matrix.cmd }} -v -st --solver-timeout-assertion 0 --solver-threads 6 --solver-command=yices-smt2
```

Comment:
i'm concerned too many parallel solver processes could actually degrade performance. we have only 4 cores in ubuntu runner. wdyt?

Reply:
you're right, back to 4